packages/tupaia-web/src/features/DashboardItem/DashboardInfoHover.tsx

```diff
@@ -3,64 +3,103 @@
+import { MOBILE_BREAKPOINT } from '../../constants';
 
-const Wrapper = styled.div`
-  text-transform: none;
-  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
-  cursor: pointer;
-  color: ${({ theme }) => theme.palette.common.white};
-  border-color: ${({ theme }) => theme.palette.common.white};
-  padding: 0.3rem;
+const DesktopWrapper = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
+  color: ${({ theme }) => theme.palette.common.white};
   flex-direction: column;
-  gap: 0.3rem;
-  margin-top: 1rem;
-  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
-    .MuiButton-label {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      align-items: center;
-    }
-    position: absolute;
-    width: 100%;
-    height: 100%;
+  position: absolute;
+  padding: 1rem;
+  width: 100%;
+  height: 100%;
+  background-color: ${({ theme }) => theme.palette.common.black};
+  top: 0;
+  left: 0;
+  opacity: 0;
+
+  svg {
+    font-size: 1.4rem;
+  }
+
+  p {
+    font-size: 0.9rem;
+  }
+
+  &:hover,
+  &:focus-visible {
+    cursor: auto;
+    opacity: 0.9;
     background-color: ${({ theme }) => theme.palette.common.black};
-    border: none;
-    top: 0;
-    left: 0;
-    opacity: 0;
-    margin-top: 0;
-    &:hover,
-    &:focus-visible {
-      opacity: 0.9;
-      background-color: ${({ theme }) => theme.palette.common.black};
-    }
+  }
+
+  @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
+    display: none;
   }
 `;
 
-const DashboardInfoHoverText = styled(Typography)`
-  font-size: 1rem;
+const MobileWrapper = styled.div`
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    display: none;
+  }
+`;
+
+const ExpandButton = styled(Button).attrs({
+  variant: 'outlined',
+  color: 'default',
+})<{
+  $active: boolean;
+}>`
+  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
+  text-transform: none;
+  color: ${({ theme }) => theme.palette.common.white};
+  border-color: ${({ theme }) => theme.palette.common.white};
+  padding: 0.3rem;
+  width: 100%;
+  margin-top: 1rem;
+
+  .MuiButton-label span {
+    display: ${props => (props.$active ? 'none' : 'flex')};
+  }
+
+  .MuiTypography-root {
+    display: ${props => (props.$active ? 'block' : 'none')};
+    padding: 0.3rem;
+  }
 `;
 
 interface DashboardInfoHoverProps {
   infoText: string | undefined;
 }
 
 export const DashboardInfoHover = ({ infoText }: DashboardInfoHoverProps) => {
+  const [isInfoBoxOpen, setIsInfoBoxOpen] = useState(false);
+
   if (!infoText) return null;
-  const content = <DashboardInfoHoverText>{infoText}</DashboardInfoHoverText>;
+
+  const toggleInfoBox = () => {
+    setIsInfoBoxOpen(!isInfoBoxOpen);
+  };
+
   return (
-    <Wrapper>
-      <MuiInfoIcon />
-      {content}
-    </Wrapper>
+    <>
+      <DesktopWrapper>
+        <MuiInfoIcon />
+        <Typography>{infoText}</Typography>
+      </DesktopWrapper>
+      <MobileWrapper>
```

Comment:
This update males the dashboard info hover into an expandable button on mobile as per the OG site